frontend/src/components/NavigationButton.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { useAttrs } from 'vue';
```

Comment:
The [Vuejs doc](https://vuejs.org/api/sfc-script-setup.html#useslots-useattrs) states that the use of `useAttrs` should be rare and I suggest we do as they say. 

What you need here is to just define the component props. 
- Declare an interface
- Use `const props = defineProps<MyInterface>()`
- Use your attributes as follow : `props.myprop`

Reply:
I should read docs more carefully ...